arisa.json

```diff
@@ -175,6 +176,27 @@
           "MCPE"
         ]
       },
+      "multiplePlatforms": {
+        "platformWhitelist": [
+          "Amazon",
+          "Android",
+          "Gear VR",
+          "iOS",
+          "Nintendo Switch",
+          "PlayStation 4",
+          "Windows 10 - PC",
+          "Windows 10 Mobile",
+          "Xbox One"
+        ],
+        "targetPlatform": ["Multiple"],
+        "transferredPlatformBlacklist": ["None"],
+        "keepPlatformTag": ["MEQS_KEEP_PLATFORM"],
+        "whitelist": ["MCPE"]
+      },
+      "keepPlatform": {
+        "keepPlatformTag": ["MEQS_KEEP_PLATFORM"],
```

Comment:
Why is this a list? 🤔

Reply:
Which part shouldn't be a list?

Reply:
keepPlatformTag

Reply:
Is it? TargetPlatform is the same...
I... Uh, got it from the Arisa.json for ConfirmParent and KeepPrivate, and they're not lists but are written that way

Reply:
keepPlatformTag": "MEQS_KEEP_PLATFORM" to make sure no one thinks we can put more than 1 (and read as String, not List<String>)

Reply:
Should I remove it from the target platform as well?

Reply:
yep

Reply:
K, removed from the targetConfirmationStatus as well